library/src/main/java/com/sothree/slidinguppanel/SlidingUpPanelLayout.kt

```diff
@@ -1140,7 +1234,13 @@ class SlidingUpPanelLayout @JvmOverloads constructor(context: Context, attrs: At
             setAllChildrenVisible()
         }
 
-        override fun onViewPositionChanged(changedView: View?, left: Int, top: Int, dx: Int, dy: Int) {
+        override fun onViewPositionChanged(
```

Comment:
This multi-line codestyle blows code up. I don't like it, but it's ok.

Reply:
I keep not that much attention why, but on my side if mostly use a single line.
I've the flavor  in my head of preventing huge files.

Reply:
was able to trim it down by a couple of line :) 
please let me know what else can be done to be able to merge this :)